sormas-ui/src/main/java/de/symeda/sormas/ui/selfreport/SelfReportController.java

```diff
@@ -124,4 +140,56 @@ public CommitDiscardWrapperComponent<SelfReportDataForm> getSelfReportEditCompon
 
 		return editComponent;
 	}
+
+	public void processSelfReport(String uuid) {
+		SelfReportDto selfReport = FacadeProvider.getSelfReportFacade().getByUuid(uuid);
+		SelfReportProcessingFLow flow = new SelfReportProcessingFLow(getSelfReportProcessingFacade());
+
+		boolean isCaseReport = selfReport.getType() == SelfReportType.CASE;
```

Comment:
if selfReport is null, will get a NPE here

Reply:
yes, but why would selefReport be null?
Most probably because of a bug, so it's good that it crashes.